x/outline-connectivity-app/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "author": "Daniel LaCosse",
+  "dependencies": {
+    "lit": "^2.7.6"
+  },
+  "description": "A basic cross-platform demo of the Outline SDK",
+  "devDependencies": {
+    "@yarnpkg/sdks": "^3.0.0-rc.48",
+    "typescript": "^5.1.6"
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "name": "outline_sdk_connectivity_test",
+  "packageManager": "yarn@3.6.1",
+  "private": true,
+  "scripts": {
+    "app_desktop": "yarn workspace app_desktop",
+    "app_mobile": "yarn workspace app_mobile",
```

Comment:
What do these commands do? Are they shortcuts to run commands in those workspaces?